precompile/contracts/noop/contract.go

```diff
@@ -0,0 +1,18 @@
+package noop
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/precompile/contract"
+)
+
+// NewContract returns a new noop stateful precompiled contract.
```

Comment:
Might be good to update
```
// NewContract returns a new noop stateful precompiled contract.
// 	This contract is used for testing purposes only and should not be used on public chains. 
//	The functions of this contract (once implemented), will be used to exercise and test the various aspects of
// 	the EVM such as gas usage, argument parsing, events, etc. The specific operations tested under this contract are
// 	still to be determined.
```
So future users are reminded with godocs

<img width="708" alt="image" src="https://github.com/user-attachments/assets/f60660e1-5cd8-4a7e-bd88-5da82252c174">

Reply:
Great reminder to document as close to the code/functions as possible and take advantage of godoc